Source/WebCore/page/cocoa/WebTextIndicatorLayer.mm

```diff
@@ -91,7 +91,7 @@ static bool indicatorWantsFadeIn(const WebCore::TextIndicator& indicator)
     return false;
 }
 
-- (void)updateWithFrame:(CGRect)frame textIndicator:(RefPtr<WebCore::TextIndicator>)textIndicator margin:(CGSize)margin offset:(CGPoint)offset updatingIndicator:(BOOL)updatingIndicator
+- (void)updateWithFrame:(CGRect)frame textIndicator:(WebCore::TextIndicator *)textIndicator margin:(CGSize)margin offset:(CGPoint)offset updatingIndicator:(BOOL)updatingIndicator
```

Comment:
I think `WebCore::TextIndicator*` is preferred, since this is not an ObjC object.

```suggestion
- (void)updateWithFrame:(CGRect)frame textIndicator:(WebCore::TextIndicator*)textIndicator margin:(CGSize)margin offset:(CGPoint)offset updatingIndicator:(BOOL)updatingIndicator
```